code/_core/obj/structure/interactive/botany/_plant.dm

```diff
@@ -96,6 +97,12 @@
 		if(move_direction & WEST)
 			animation_offset_x += 32
 
+		//Harvester's botany skill increases the harvested results based on their skill power.
+		//Each point in skill power add .2 potency and .1 yield.
+		var/skillPower = caller.get_skill_power(SKILL_BOTANY)
+		potency = potency + (potency * skillPower * 2)
+		yield += yield * skillPower
```

Comment:
Not sure if this is intended, but with each harvest, the yield and potency of the plant would increase. This wouldn't just affect the current harvest, but future harvests as well.

Reply:
I'll update the after-harvest code to reset these back to the initial values, instead of (initial + current)/2